src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,49 @@
+package lotto.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Lotto {
+
+    private static final int START_RANGE = 1;
+    private static final int END_RANGE = 45;
+    private static final int COUNT = 6;
+    private final List<Integer> lottoNumbers;
```

Comment:
상수와 인스턴스 변수는 한 줄 띄어서 가독성을 높여주세요!🙏 
```suggestion
    private static final int COUNT = 6;
    
    private final List<Integer> lottoNumbers;
```